packages/ui/src/svg.d.ts

```diff
@@ -1,6 +1,5 @@
 declare module '*.svg' {
-  const content: React.FC<
-    React.SVGAttributes<SVGElement> & React.RefAttributes
-  >;
+  import { SvgProps } from 'react-native-svg';
```

Comment:
Is this necessary? I had tried to set it up so that the shared package used the more generic type and `svg.d.ts` in tlon mobile overrode it with the native types.

Reply:
Looks like it's not